README-Networking.md

```diff
@@ -33,9 +39,32 @@ CNI-Genie is a CNI plugin that enables Kubernetes to simultaneously have access
   - Service Discovery
     - micro DNS at each node, DNS lookups always local
   - Multicast Networking - UDP multcast even if underlying network doesn't support it
-  - 
+
 **Flannel**
 
+Try it out: `kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml`
+  - Focused on networking. For network policy, other projects such as Calico can be used
```

Comment:
Remove "Focused on networking" - implied

Reply:
Nevermind, Flannel needs another component for its CNI it looks like, https://github.com/coreos/flannel-cni